build_sysext

```
@@ -0,0 +1,107 @@
+
+SCRIPT_ROOT=$(dirname "$(readlink -f "$0")")
+. "${SCRIPT_ROOT}/common.sh" || exit 1
+
+# Script must run inside the chroot
+assert_inside_chroot
+assert_root_user
+
+# All these are used to set up the 'BUILD_DIR' variable
+DEFINE_string board "${DEFAULT_BOARD}" \
+  "The board to build a sysext for."
+DEFINE_string output_root "${DEFAULT_BUILD_ROOT}/images" \
+  "Directory in which to place sysext images and intermediate files"
+DEFINE_string group "developer" \
+  "The normal image build group"
+DEFINE_integer build_attempt 1 \
+  "The build attempt for this build."
+
+FLAGS_HELP="USAGE: build_sysext [flags] [sysext name] [packages to build].
+This script is used to build a Flatcar sysext. 
+
+Examples:
+
+Build a systemd-sysext image named command-extras with package_1, package_2 and package_3 for amd64:
+sudo build_sysext --board=amd64-usr command-extras package_1 package_2 package_3
+
+Builds a systemd-sysext image named command-extras with package_1, package_2 and package_3 for arm64:
+sudo build_sysext --board=arm64-usr command-extras package_1 package_2 package_3
+
+"
+
+show_help_if_requested "$@"
+
+# Parse command line
+FLAGS "$@" || exit 1
+
+eval set -- "${FLAGS_ARGV}"
+
+. "${BUILD_LIBRARY_DIR}/release_util.sh" || exit 1
+
+. "${BUILD_LIBRARY_DIR}/toolchain_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/board_options.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/build_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/prod_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/dev_container_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/test_image_content.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/torcx_manifest.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/vm_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/cros_vm_constants.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/oem_sysext_util.sh" || exit 1
+
+set -euo pipefail
+
+cleanup() {
+  umount "${BUILD_DIR}/fs-root" "${BUILD_DIR}/install-root" "${BUILD_DIR}/workdir" || true
```

```suggestion
  umount "${BUILD_DIR}/fs-root" "${BUILD_DIR}/install-root" "${BUILD_DIR}/workdir" 2>/dev/null || true
```
Suppress error output to have clearer logs